index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sameed - Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+</head>
+
+
+
+<body>
+
+    <header>
+        <nav>
+            <div class="logo-container">
+                <a class="logo-text">Sameed Mohsin</a>
+            </div>
+
+            <div class="menu">
+                <img src="./Images/Hamburger.svg" alt="" class="hamburger">
+            </div>
```

Comment:
- [ ] [OPTIONAL] The size of your hamburger button is too large, please resize it to fit the Figma template design.
- [ ] Your toolbar is too tall, please reduce the position-top of 44px on the nav element CSS declarations.

Figma Design
![image](https://user-images.githubusercontent.com/84156714/212926898-45aa5761-608a-4de7-a646-570927198e14.png)

Your work
![image](https://user-images.githubusercontent.com/84156714/212926991-a6c04690-69e4-414e-9ba6-a8ffb6dc07f5.png)